### DISCUSSION TITLE
Loading scripts from latest is deprecated and will be removed soon.

### DISCUSSION THREAD

**Discussion by GoToLoop** [pyscript/pyscript#1920] [Proposals]
Please, don't remove loading latest "core.js" & "core.css" from https://PyScript.net/latest/ 🥺 

https://PyScript.com/@gotoloop/unicode-letters-test/latest

**Comment by GoToLoop** [comment]
Found a workaround for latest version via https://cdn.JsDelivr.net/npm/@pyscript/core.
However I still think https://PyScript.net/latest/core.js has no good reason to be removed or deprecated.

P.S.: An even shorter CDN link: https://Unpkg.com/@pyscript/core

**Reply by WebReflection** [reply]
please be aware what lands in *npm* is meant to be unstable (canary or dev channel) and we still suggest people to use official releases and their pinned version to avoid surprises.

**Reply by GoToLoop** [reply]
So allow us to get the latest official stable version!

I'd hate to have to keep updating the "index.html" file from old projects all the time just to keep them getting latest!

**Reply by WebReflection** [reply]
Pinning versions of any software is usually a better practice than relying "whatever latest" ... and because latter approach can break the Web, we encourage pinned versions instead so that if something changes and/or breaks, you won't need to ever worry about it.

However, if you do want the latest, considering we still have tons of tests so we tend not to publish code that breaks, you can use the dev channel as "whatever latest".

That is:

  * for the JS module: https://cdn.jsdelivr.net/npm/@pyscript/core/dist/core.js
  * for the CSS file: https://cdn.jsdelivr.net/npm/@pyscript/core/dist/core.css

We removed *latest* for the mentioned reasons so I hope this alternative approach at least gives you a way to leave at the edge of the project.
